.travis.yml

```diff
@@ -28,13 +26,12 @@ matrix:
       sudo: required
       language: generic
 install:
-  - pip install pipenv==11.10
-  - pipenv install --three --dev Pipfile
+  - pip3 install .[dev]
 script:
-  - pipenv run pytest
-  - if [[ "$RUN_MYPY" == "yes" ]]; then pipenv run pip install -r requirements.txt && pipenv run ./tools/run-mypy; fi
-  - if [[ "$RUN_ISORT_CHECK" == "yes" ]]; then pipenv run pip install -r requirements.txt && pipenv run ./tools/run-isort-check; fi
-  - if [[ "$RUN_PYCODESTYLE" == "yes" ]]; then pipenv run pip install -r requirements.txt && pipenv run pycodestyle; fi
+  - pytest
+  - if [[ "$RUN_MYPY" == "yes" ]]; then pip install -r requirements.txt && ./tools/run-mypy; fi
```

Comment:
Why is this line `pip` instead of `pip3`?

Reply:
Oh, this was fixed in the second commit. But that's confusing, should have been done in first commit.

Reply:
Right; I missed this in trying to make clear the progression of changes. Will be updated in the next push.